azure/packages/azure-client/src/interfaces.ts

```diff
@@ -56,28 +56,66 @@ export interface AzureGetVersionsOptions {
     maxCount: number;
 }
 
+export type AzureConnectionConfigType = "local" | "remote";
```

Comment:
Any relevant documentation we should add to this?

Reply:
I just added a docstring for this 👍 
Also it's explained in the README.md